Sources/Hedera/Contract/ContractFunctionResult.swift

```diff
@@ -69,6 +69,10 @@ public struct ContractFunctionResult {
     /// This is always empty in a ContractLocalCallQuery response, since no internal creations can happen in a static EVM call.
     public let contractNonces: [ContractNonceInfo]
 
+    /// If not null this field specifies what the value of the signer account nonce is post transaction execution.
+    /// For transactions that don't update the signer nonce (like HAPI ContractCall and ContractCreate transactions) this field should be null.
+    public let mSignerNonce: UInt64?
```

Comment:
```suggestion
    public let signerNonce: UInt64?
```
?

Reply:
Different naming conventions were proposed but it's still not consistent between SDK to SDK. Best to deal with this later. Getting rid of `m`.